src/backend/electron/main.ts

```diff
@@ -543,36 +485,13 @@ registerIpcMainHandle<IpcMainHandle>({
     };
   },
 
-  GET_HOW_TO_USE_TEXT: () => {
-    return howToUseText;
-  },
-
-  GET_POLICY_TEXT: () => {
-    return policyText;
-  },
-
-  GET_OSS_LICENSES: () => {
-    return ossLicenses;
-  },
-
-  GET_UPDATE_INFOS: () => {
-    return updateInfos;
-  },
-
-  GET_OSS_COMMUNITY_INFOS: () => {
-    return ossCommunityInfos;
-  },
-
-  GET_CONTACT_TEXT: () => {
-    return contactText;
-  },
-
-  GET_Q_AND_A_TEXT: () => {
-    return qAndAText;
-  },
-
-  GET_PRIVACY_POLICY_TEXT: () => {
-    return privacyPolicyText;
+  GET_ASSET_TEXT: async (_, textType) => {
+    const fileName = path.join(__static, AssetTextFileNames[textType]);
+    const text = fs.promises.readFile(fileName, "utf-8");
+    if (textType === "OssLicenses" || textType === "UpdateInfos") {
+      return JSON.parse(await text) as AssetObjectType[typeof textType];
```

Comment:
ここで`AssetObjectType`を使っていますが、まあひとまとめにしちゃってるのでこれでも良いはず！
```suggestion
      return JSON.parse(text) as TextAsset[typeof textType];
```

オブジェクトのんとstringのん分けたい気持ちもあるけど、まあ別にこれで問題はなさそう･･･！！！

Reply:
修正しました。